webapp/channels/src/components/global_header/left_controls/product_menu/product_menu.tsx

```diff
@@ -140,7 +142,13 @@ const ProductMenu = (): JSX.Element => {
                         aria-controls='product-switcher-menu'
                     />
                     {license.IsLicensed === 'false' && <ProductBrandingTeamEdition/>}
-                    {license.IsLicensed === 'true' && <ProductBranding/>}
+                    {license.IsLicensed === 'true' && EnableCustomBrand === 'true' && CustomBrandHasLogo === 'true' && (
+                        <img
+                            src={Client4.getCustomDarkLogoUrl('0')}
+                            height={30}
```

Comment:
update to 24 so there is a bit more breathing room in the header

Reply:
Done